test/auto-list.test.js

```diff
@@ -409,6 +409,41 @@ describe('List aggregator (for an auto list)', function () {
     )
   })
 
+  it('should ignore any specific list items', function (done) {
```

Comment:
Hi @benjaminparnell 

I think this needs to be a little more verbose and really explain what is going on.